modules/apps/headless/headless-asset-library/headless-asset-library-impl/src/main/java/com/liferay/headless/asset/library/internal/resource/v1_0/AssetLibraryResourceImpl.java

```diff
@@ -249,11 +250,10 @@ public AssetLibrary patchAssetLibrary(
 			_addOrUpdateDepotEntry(
 				assetLibrary,
 				LocalizedMapUtil.getLocalizedMap(
-					contextAcceptLanguage.getPreferredLocale(), description,
-					descriptionMap),
+					LocaleUtil.getDefault(), description, descriptionMap),
```

Comment:
This way you can not create the asset with the api locale, so I do not think this is the way